src/components/pages/accounts/NetWorthPie.tsx

```diff
@@ -20,26 +20,8 @@ export default function NetWorthPie({
   const { data: currency } = API.useMainCurrency();
   const unit = currency?.mnemonic || '';
 
-  let assetsTotal = 0;
-  let liabilitiesTotal = 0;
-  assetsTotal = Object.entries(assetsSeries || {}).reduce(
-    (total, [monthYear, amount]) => {
-      if (DateTime.fromFormat(monthYear, 'MM/yyyy') <= selectedDate) {
-        return total.add(amount);
-      }
-      return total;
-    },
-    new Money(0, unit),
-  ).toNumber();
-  liabilitiesTotal = Object.entries(liabilitiesSeries || {}).reduce(
-    (total, [monthYear, amount]) => {
-      if (DateTime.fromFormat(monthYear, 'MM/yyyy') <= selectedDate) {
-        return total.add(amount);
-      }
-      return total;
-    },
-    new Money(0, unit),
-  ).toNumber() * -1;
+  const assetsTotal = assetsSeries?.[selectedDate.toFormat('MM/yyyy')] || new Money(0, unit);
+  const liabilitiesTotal = liabilitiesSeries?.[selectedDate.toFormat('MM/yyyy')] || new Money(0, unit);
```

Comment:
no need to calculate totals as now this series have the total already calculated